minicluster/src/main/java/alluxio/multi/process/MultiProcessCluster.java

```diff
@@ -469,9 +480,9 @@ private synchronized Worker createWorker(int i) throws IOException {
     File ramdisk = new File(mWorkDir, "ramdisk" + i);
     logsDir.mkdirs();
     ramdisk.mkdirs();
-    int rpcPort = PortUtils.getFreePort();
-    int dataPort = PortUtils.getFreePort();
-    int webPort = PortUtils.getFreePort();
+    int rpcPort = getNewPort();
+    int dataPort = getNewPort();
+    int webPort = getNewPort();
```

Comment:
Should `createWorker` return the same ports for the same index instead of making new ones each time?

Reply:
the method is private and called once in the constructor, so there will only be one call and the ports for each worker will be the same across runs

Reply:
sorry, it's actually used from `start()`, which is public. But the start method is only allowed to be called once, so we should be ok

Reply:
got it, thanks.